dracut-bash-completion.sh

```diff
@@ -15,83 +15,92 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-__contains_word () {
-        local word="$1"; shift
-        for w in "$@"; do [[ $w = "$word" ]] && return 0; done
-        return 1
+__contains_word() {
+    local word="$1"
```

Comment:
Is the removal of the semicolon here ```word="$1"``` from the previous statement ```word="$1";``` intentional or an oversight?

Reply:
a semicolon at the end of a line is redundant in shell